Octokit.Tests.Integration/Clients/RepositoryInvitationsClientTests.cs

```diff
@@ -0,0 +1,111 @@
+                Assert.Equal(invitations[0].Permissions, response.Permissions);
+                Assert.Equal(invitations[0].Repository.Id, response.Repository.Id);
+            }
+        }
+    }
+
+    public class TheGetAllForCurrentMethod
+    {
+        [IntegrationTest]
+        public async Task CanGetAllInvitations()
+        {
+            var github = Helper.GetAuthenticatedClient();
+            var repoName = Helper.MakeNameWithTimestamp("public-repo");
+
+            using (var context = await github.CreateRepositoryContext(new NewRepository(repoName)))
+            {
+                var fixture = github.Repository.Collaborator;
+                var permission = new CollaboratorRequest(Permission.Push);
+
+                // invite a collaborator
+                var response = await fixture.Invite(context.RepositoryOwner, context.RepositoryName, context.RepositoryOwner, permission);
+
+                Assert.Equal(context.RepositoryOwner, response.Invitee.Login);
+                Assert.Equal(InvitationPermissionType.Write, response.Permissions);
+
+                var invitations = await github.Repository.Invitation.GetAllForCurrent();
+
+                Assert.True(invitations.Count >= 1);
+                Assert.NotNull(invitations.FirstOrDefault(i => i.CreatedAt == response.CreatedAt));
+                Assert.NotNull(invitations.FirstOrDefault(i => i.Id == response.Id));
+                Assert.NotNull(invitations.FirstOrDefault(i => i.Inviter.Login == response.Inviter.Login));
+                Assert.NotNull(invitations.FirstOrDefault(i => i.Invitee.Login == response.Invitee.Login));
+                Assert.NotNull(invitations.FirstOrDefault(i => i.Permissions == response.Permissions));
+                Assert.NotNull(invitations.FirstOrDefault(i => i.Repository.Id == response.Repository.Id));                
+            }
+        }
+    }
+
+    public class TheDeclineMethod
+    {
+        [IntegrationTest]
+        public async Task CanDeclineInvitation()
+        {
+            var github = Helper.GetAuthenticatedClient();
+            var repoName = Helper.MakeNameWithTimestamp("public-repo");
+
+            using (var context = await github.CreateRepositoryContext(new NewRepository(repoName)))
+            {
+                var fixture = github.Repository.Collaborator;
+                var permission = new CollaboratorRequest(Permission.Push);
+
+                // invite a collaborator
+                var response = await fixture.Invite(context.RepositoryOwner, context.RepositoryName, context.RepositoryOwner);
+
+                Assert.Equal(context.RepositoryOwner, response.Invitee.Login);
+                Assert.Equal(InvitationPermissionType.Write, response.Permissions);
+
+                var declineResponse = await github.Repository.Invitation.Decline(response.Id);
+
+                Assert.True(declineResponse);
+
+                //Assert.True(invitations.Count >= 1);
```

Comment:
> Any ideas?

I'm happy to punt on the receiving end of the request and leave an open issue to track that, because we don't really have the infrastructure right now to support multiple accounts...